docassemble/AssemblyLine/al_document.py

```diff
@@ -1444,12 +1483,22 @@ def init(self, *pargs, **kwargs):
         if not hasattr(self, "auto_labeler"):
             self.auto_labeler = alpha
         if not hasattr(self, "auto_ocr"):
-            self.auto_ocr = True
+            self.auto_ocr = False
+        if not hasattr(self, "include_table_of_contents"):
+            self.include_table_of_contents = True
+        if not hasattr(self, "include_exhibit_cover_pages"):
+            self.include_exhibit_cover_pages = True
+        if not hasattr(self, "bates_prefix"):
+            self.bates_prefix = ""
         self.object_type = ALExhibit
         self.complete_attribute = "complete"
 
     def as_pdf(
-        self, filename="file.pdf", pdfa: bool = False, add_cover_pages: bool = True
```

Comment:
Fairly sure the `add_cover_pages` parameter is in use.

Reply:
Github search only shows this file as having it.

As I mentioned above, IMO the only public API (at least the only API that we should actively use and expect people to use) is `ALExhibitDocument`, and maybe the fact that `ALExhibitList` is a `DAList`.

In general I was trying to reducing the duplicate of params. I moved params to objects-block / initialization only if I thought that they wouldn't change dynamically based on the interview.